### ISSUE TITLE
Suggested addition: Hover preview box back to normal page

### ISSUE THREAD

**Issue by MILKMILKKING** [nothingislost/obsidian-hover-editor#13]
Suggested addition: Hover preview box back to normal page, suggest to put this option in "More settings" at the top right of the window.
The good thing is that now I can hover or hover specific pages by mouse, but sometimes I have to restore the hover window to normal page for viewing, currently I can only close the hover window and reopen the file. If you add a new option, it will be easier to restore it quickly.

**Comment by nothingislost** [comment]
Sorry, I'm not sure I'm following what you're asking for here. Could you try and restate your request?

**Comment by MILKMILKKING** [comment]
Sorry I didn't express myself clearly, I'll try to describe it again (it might be a bit wordy).
For example I can now hover over this one note.
![image](https://user-images.githubusercontent.com/69726213/158407455-98129df5-7f68-4095-9068-64f971984ceb.png)

But now I want to revert this open hover window to a normally displayed page, but I am currently unable to do so:
![image](https://user-images.githubusercontent.com/69726213/158407486-927af522-698a-4715-9262-4a1bb5389144.png)

Since the hovered note does not appear in the open record, sometimes to open this note, I need to look for this note in the file list in order to reopen it.
Thanks for the reply, it's a great plugin!

**Comment by nothingislost** [comment]
Thanks for the additional detail. So you would like to be able to create a popover and have the option drag it back into the main workspace? We're working on this capability at the moment which I believe should give you the functionality you're looking for.

**Comment by nothingislost** [comment]
Delivered in 0.6.0
